packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -109,4 +117,20 @@ export default class VotesRepo extends AbstractRepo<DbVote> {
       WHERE ARRAY_POSITION("authorIds", $1) IS NOT NULL
     `, [oldUserId, newUserId]);
   }
+  
+  async getDigestPlannerVotesForPosts(postIds: string[]): Promise<Array<PostVoteCounts>> {
+    return await logIfSlow(async () => await this.getRawDb().manyOrNone(`
+      SELECT p._id as "postId",
+        count(v._id) FILTER(WHERE v."voteType" = 'smallUpvote') as "smallUpvoteCount",
+        count(v._id) FILTER(WHERE v."voteType" = 'bigUpvote') as "bigUpvoteCount",
+        count(v._id) FILTER(WHERE v."voteType" = 'smallDownvote') as "smallDownvoteCount",
+        count(v._id) FILTER(WHERE v."voteType" = 'bigDownvote') as "bigDownvoteCount"
+      FROM "Posts" p
+      JOIN "Votes" v tablesample system(50) ON v."documentId" = p."_id"
+      WHERE p._id = ANY($1)
+        AND v."collectionName" = 'Posts'
+        AND v.cancelled = false
+      GROUP BY p._id
+    `, [postIds]), "getDigestPlannerVotesForPosts");
+  }
```

Comment:
This is not currently used, but I plan to finish this part and add it to the table once I discuss with Lizka